app/views/shared/_email_languages.html.erb

```diff
@@ -29,6 +29,7 @@ locals:
           ),
           locale,
           checked: selection ? selection.to_s == locale.to_s : I18n.locale.to_s == locale.to_s,
+          lang: locale,
```

Comment:
We have `lang` above on line 28 as well. Do we need both?

Reply:
Line 32 puts the attribute on the element while 28 puts it on the `<span>` attribute itself. Right now, if a user activates the read-all feature, it will read both the input and the text in the set languages. 

The fact that this is happening should be discussed further and that is why I wrote [LG-9949](https://cm-jira.usa.gov/browse/LG-9949). There have been [previous discussions](https://gsa-tts.slack.com/archives/C01710KMYUB/p1646230434545949) on whether or not similar behaviors are priority fixes. So for now, I think we both.